docs/concepts/privacy.md

```diff
@@ -0,0 +1,63 @@
+
+This is one of the advantages of IPFS over traditional legacy-web hosting. It means retrieving files — especially popular ones that exist on lots of nodes in the network — can be faster and more bandwidth-efficient. However, it's important to note that those DHT queries happen in public. Because of this, it's possible that third parties could be monitoring this traffic to determine what CIDs are being requested, when, and by whom. As IPFS continues to grow in popularity, it's more likely that such monitoring will exist.
+
+### Node IDs and node requests
+
+The other half of the equation when considering the prospect of IPFS traffic monitoring is that nodes' unique identifiers are themselves public. Just like with CIDs, every individual IPFS node has its own public identifier, such as `QmRGgYP1P5bjgapLaShMVhGMSwGN9SfYG3CM2TfhpJ3igE`.
+
+While a long string of letters and numbers may not be a "Johnny Appleseed" level of human-readable specificity, it's still a long-lived, unique identifier for your node. Keep in mind that it's possible to do a DHT lookup on your node's identifier and, particularly if your node is regularly running from the same location (like your home), find your IP address. Additionally, longer-term monitoring of the public IPFS network could yield information about what CIDs your node is requesting and/or reproviding and when.
+
+## Enhancing your IPFS privacy
+
+If there are situations in which you know you'll need to remain private but still want to use IPFS, one of the approaches outlined below may help. And don't forget, you can always discuss privacy and get others' input or ideas in the official [IPFS forums](https://discuss.ipfs.io).
+
+### Using a public gateway
+
+Using a public [IPFS gateway](https://dweb-primer.ipfs.io/avenues-for-access/review-these-lessons-from-the-tutorial-on-interacting-with-the-classical-http-web/public-gateways) is one way to request IPFS-hosted content without revealing any information about your local node — because you aren't using a local node! However, this method does keep you from enjoying all the benefits of being a full participant in the IPFS network.
```

Comment:
Would internal link be better? 
We have bit more up to date section related to http gateways that includes basic intro to subdomain ones:

```suggestion
Using a public [IPFS gateway](/how-to/address-ipfs-on-web/#http-gateways) is one way to request IPFS-hosted content without revealing any information about your local node — because you aren't using a local node! However, this method does keep you from enjoying all the benefits of being a full participant in the IPFS network.
```

Reply:
Was going back and forth on this one -- I like that the dweb primer example is novice-friendly, but the docs example does mention subdomain gateways, which is important. We're also providing a handful of other dweb primer links, which does provide a path for the more novice readers. Good call.